lib/git_reflow/git_server/git_hub/pull_request.rb

```diff
@@ -63,7 +63,11 @@ def comments
         end
 
         def last_comment
-          "#{comments.last.body.inspect}"
+          if comments.nil? or comments.last.nil?
```

Comment:
Discussed this further with @simonzhu24 and we agreed to just add a spec for the `comments` method to ensure it always returns a string and remove the extra nil check.